test/testsuite/Spec/Random.hs

```diff
@@ -0,0 +1,18 @@
+module Spec.Random (spec) where
+
+import qualified RIO.ByteString as BS
+import           Test.Hspec as Hspec
+
+-- ⚛
```

Comment:
What does this symbol mean?

Reply:
It's the atom symbol, under which I place all the Fission stuff.

Reply:
Ooooooooh haha I see. Fission... atomic... yeah cool cool 👍😆

I like your puns

Reply:
![](https://media1.giphy.com/media/Qem49y6xGw57W/giphy.gif?cid=36b14fac38562ee5380174fc825a6f1f0fd0627cb4987334&rid=giphy.gif)